opensciencegrid/vo-frontend/Dockerfile

```
@@ -11,18 +11,17 @@ LABEL maintainer OSG Software <help@opensciencegrid.org>
 RUN groupadd -o -g 993 frontend
 RUN useradd -o -u 996 -g 993 -s /bin/sh frontend
 
-RUN yum -y install \
-    git \
-    jq \
-    patch \
-    python3-scitokens \
-    rsyslog \
-    rsyslog-gnutls \
-    vo-client
-
-# pull gwms from upcoming
-RUN yum -y install \
-           glideinwms-vofrontend-standalone
+RUN yum -y clean all && \
+    yum -y install \
+      git \
+      glideinwms-vofrontend-standalone \
+      jq \
+      patch \
+      python3-scitokens \
+      rsyslog \
+      rsyslog-gnutls \
+      vim \
+      vo-client
```

```suggestion
RUN yum -y install \
      git \
      glideinwms-vofrontend-standalone \
      jq \
      patch \
      python3-scitokens \
      rsyslog \
      rsyslog-gnutls \
      vim \
      vo-client && \
    yum -y clean all
```
yum clean all should be after any yum installs, so it cleans up downloaded repo metadata.